src/core/gltf2importer/gltf2parser.cpp

```diff
@@ -139,8 +141,14 @@ GLTF2Parser::~GLTF2Parser()
 {
 }
 
+void msgh(QtMsgType, const QMessageLogContext &, const QString &s)
```

Comment:
sorry about this, I'm keeping it until it's not a WIP anymore because it's useful to break on whatever debug message QObject spouts